check_sites.py

```diff
@@ -18,33 +18,48 @@
 import os
 from publicsuffix2 import PublicSuffixList
 
+def find_diff_sets(old_sets, new_sets):
+    diff_sets = {}
+    subtracted_sets = {}
+    difference = set(old_sets)^set(new_sets)
+    for diff in difference:
+        if diff in new_sets:
+            diff_sets[diff] = new_sets[diff]
+        else:
+            subtracted_sets[diff] = old_sets[diff]
+    return diff_sets, subtracted_sets
```

Comment:
It looks like neither callsite (prod, test) use the second return value here. Can we remove it?

Reply:
I can remove but I wanted the removed sets variable to be a part of the return statement since we're going to need it anyway when we finalize the policy on removed sets/elements, which I was hoping would happen soon.

Reply:
I'm fine with keeping it as long as we add tests for it :)

Reply:
Added more tests now